test/worlds/gpu_lidar.sdf

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" ?>
+<!--
+  Note: For now, this file must be loaded from ign-gazebo's root folder so the
+  mesh's relative path is correct.
+-->
+<sdf version="1.6">
+  <world name="gpu_lidar_sensor">
+    <physics name="1ms" type="ode">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="ignition-gazebo-physics-system"
+      name="ignition::gazebo::systems::Physics">
+    </plugin>
+    <!--
+    <plugin
+      filename="ignition-gazebo-sensors-system"
+      name="ignition::gazebo::systems::Sensors">
+      <render_engine>ogre2</render_engine>
+    </plugin>
```

Comment:
A GPU lidar won't work without the sensors system, so I think this file should include it even though it's not necessary for the test that's currently using it (`ModelCommandAPI`). Conversely, I think `Physics` and `SceneBroadcaster` can be removed.

Reply:
Including the sensor system causes a segfault in the test and it's not needed for the test.

Reply:
Oh that sounds like a problem. Maybe not this PR's fault

Reply:
It happens when the whole test is torn down. I think it's related to there already being an RGBD sensor test that uses the sensor system.